src/components/PersonDetail/Timeline/Post/Post.tsx

```diff
@@ -61,7 +62,13 @@ export function PersonDetailTimelinePost({ timeline }: Props): ReactElement {
               <Typography variant="overline" display="block" fontWeight="bold">
                 {getTitle(key, timeline.fullDefinition.fields)}
               </Typography>
-              {isString(value) && <Typography>{value}</Typography>}
+              {isString(value) && (
+                <Typography>
+                  {dayjs(value, 'YYYY-MM-DD[T]HH:mm:ss.SSS[Z]', true).isValid()
+                    ? dayjs(value).format('dddd MMMM D, YYYY')
```

Comment:
I think it would actually more readable as `dddd D MMMM, YYYY` so it would come out something like, Thursday 9 March, 2023. Thoughts?

Reply:
ah ok, that's an easy fix.